libs/marshalling/algebra/include/nil/crypto3/marshalling/algebra/processing/curve_element.hpp

```diff
@@ -680,7 +678,7 @@ namespace nil {
                         constexpr static const chunk_type S_bit = 0x20;
 
                         if (m_unit & I_bit) {
-                            BOOST_ASSERT(iter + sizeof_field_element_chunks_count ==
+                            BOOST_VERIFY(iter + sizeof_field_element_chunks_count ==
```

Comment:
Why do we need this change? Doesn't look like this expression has a side effect.

Reply:
This is to make compiler happy about unused variable assignment.